src/framework/shortcuts/qml/MuseScore/Shortcuts/EditShortcutDialog.qml

```diff
@@ -68,13 +68,30 @@ Dialog {
 
             spacing: 20
 
+            RowLayout {
+                anchors.horizontalCenter: parent.horizontalCenter
+                spacing: 8
+
+                StyledIconLabel {
+                    iconCode: model.currentShortcut["icon"]
+                    font: ui.theme.iconsFont
+
+                    visible: !isEmpty
+                }
+
+                StyledTextLabel {
+                    text: model.currentShortcut["title"]
+                    font: ui.theme.bodyBoldFont
+
+                    visible: !isEmpty
+                }
+            }
+
             StyledTextLabel {
                 anchors.horizontalCenter: parent.horizontalCenter
 
                 width: parent.width
-                text: qsTrc("shortcuts", "Define keyboard shortcut")
-                horizontalAlignment: Text.AlignLeft
-                font:ui.theme.headerBoldFont
+                text: qsTrc("shortcuts", "Press a key or sequence of keys to assign a shortcut to this function.")
```

Comment:
Sure about the trailing fullstop?

Reply:
I was following the designs in the issue description. Perhaps it should have been removed.

> BTW: that merge commit is not supposed to be part of this PR, better rebase using something like git pull --rebase upstream master; git push -f

Yeah, sorry about that. I was kind of pressed for time earlier today when I went through the merge conflict.